### DISCUSSION TITLE
Ask (Quark)us anything No. 4!

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#26829] [Quarkus Blog/Website]
# Ask (Quark)us anything No. 4!

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-insights-qanda4/

**Comment by geojaya** [comment]
Can the team  show a short demo on configuring  VS code on Windows for creating using Quarkus  extension to create native binary/jar.  Thanking in advance

**Comment by preslavrachev** [comment]
What are your plans for supporting full-stack Web application development? I understand that the focus is on Kubernetes and containers, but there are thousands of small teams out there (like ours), planning to use Quarkus as a direct replacement for Spring Boot. I have seen quite some interesting development in this direction through Renarde and Quinoa, but I was wondering if those projects will ever get 1st-party framework adoption.

Could you elaborate on that?

**Comment by hamburml** [comment]
Could you show how a native x86 quarkus image is build via graalvm on an m1 arm chip? Thanks!
